bootstrap.sh

```diff
@@ -31,9 +58,9 @@ if [ ! -f killrvideo_bootstrapped ]; then
   # If requested, create a new superuser to replace the default superuser
   if [ "$KILLRVIDEO_CREATE_ADMIN_USER" = 'true' ]; then
     echo "=> Creating new superuser $KILLRVIDEO_ADMIN_USERNAME"
-    cqlsh $dse_ip 9042 -u $admin_user -p $admin_password -e "CREATE ROLE $KILLRVIDEO_ADMIN_USERNAME with SUPERUSER = true and LOGIN = true and PASSWORD = '$KILLRVIDEO_ADMIN_PASSWORD'"
+    cqlsh $dse_ip 9042 -u $admin_user -p $admin_password $dse_ssl $dse_request_timeout -e "CREATE ROLE $KILLRVIDEO_ADMIN_USERNAME with SUPERUSER = true and LOGIN = true and PASSWORD = '$KILLRVIDEO_ADMIN_PASSWORD'"
     # Login as new superuser to delete default superuser (cassandra)
-    cqlsh $dse_ip 9042 -u $KILLRVIDEO_ADMIN_USERNAME -p $KILLRVIDEO_ADMIN_PASSWORD -e "DROP ROLE $admin_user"
+    cqlsh $dse_ip 9042 -u $KILLRVIDEO_ADMIN_USERNAME -p $dse_ssl $dse_request_timeout $KILLRVIDEO_ADMIN_PASSWORD -e "DROP ROLE $admin_user"
```

Comment:
This looks like a bug: 
-p $dse_ssl $dse_request_timeout $KILLRVIDEO_ADMIN_PASSWORD

Shouldn't it be 
-p $KILLRVIDEO_ADMIN_PASSWORD $dse_ssl $dse_request_timeout

Reply:
Oh, nice find.  Let me fix that up. Interestingly, it didn't break. Maybe cqlsh doesn't process $dse_ssl and $dse_request_timeout as the password because they are clearly params and not the value for -p.